### PR TITLE
updates for wdi14

### PR DESCRIPTION
**Pull request by amaseda**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,23 +4,23 @@
...
 
-Describe the differences between a SQL and NoSQL DB, and when you might use each.
+Describe the differences between a SQL and NoSQL database, and when you might use each.
 
 ```text
 Your answer...
-
 ```
 
 ### Question #2
```
Comment (nayana487) [inline README.md:14]: `...Mongoose code...`

File: README.md [hunk 2]
```diff
@@ -83,7 +84,7 @@ var app = express();
 Describe the differences between Express and Rails as backend frameworks.
 
 ```text
-
+Your answer...
 ```
 
 ### Question #8
```
Comment (nayana487) [inline README.md:90]: maybe explicitly ask what is body-parser and what does it do?

Comment (nayana487) [reply README.md:90]: could be mean and ask what the equivalent is in Rails

Comment (amaseda) [reply README.md:90]: What is the equivalent in Rails?

Comment (amaseda) [reply README.md:90]: Also I think that in explaining the following lines they'll have to say what body parser does. Don't know how else they could correctly give the answer.

File: README.md (hunk 1 above)
Comment (amaseda) [reply README.md:14]: :+1:

**Pull request merged**